xray/commands/audit/sca/java/deptreemanager_test.go

```diff
@@ -53,12 +57,12 @@ func TestGetGraphFromDepTree(t *testing.T) {
 		},
 	}
 
-	manager := &depTreeManager{}
+	manager := &gradleDepTreeManager{&DepTreeManager{}}
 	outputFileContent, err := manager.runGradleDepTree()
 	assert.NoError(t, err)
 	depTree, uniqueDeps, err := getGraphFromDepTree(outputFileContent)
 	assert.NoError(t, err)
-	assert.ElementsMatch(t, uniqueDeps, testCase.expectedUniqueDeps, "First is actual, Second is Expected")
+	reflect.DeepEqual(uniqueDeps, testCase.expectedUniqueDeps)
 
 	for _, dependency := range depTree {
 		depChild, exists := testCase.expectedTree[dependency.Id]
```

Comment:
Let's remove the repeated GavPackageTypeIdentifier for the dependencies
```suggestion
		dependencyId := strings.TrimPrefix(dependency.Id, GavPackageTypeIdentifier)
		depChild, exists := testCase.expectedTree[dependencyId]
```